backend/src/datastore/PostDataStore.js

```diff
@@ -1,88 +1,128 @@
 const {DataSource} = require('apollo-datasource');
 const {UserInputError, AuthenticationError} = require('apollo-server-errors');
+const neode = require('../database/NeodeConfiguration');
+const {User} = require('../datastore/UserDataStore');
 
 class Post {
-    constructor(id, title, author_id, votes = new Map()) {
+    constructor({id, title, author, node = null}) {
```

Comment:
Is it possible to do it somehow without a null initialisation? It would be a bit more cleaner